frontend/front/src/pages/Admin/home/SurveyVisitProfile.js

```diff
@@ -71,9 +73,9 @@ const SurveyProfile = () => {
       </Typography>
       {surveyVisit && houseData ? (
         <AdminSurvey
-          defaultData={surveyVisit.responses}
+          defaultData={surveyAnswers}
           activeHome={houseData}
-          surveyId={surveyVisit.surveyId}
+          surveyId={surveyVisit.id}
```

Comment:
`surveyVisit.id` is the ID of the survey_visit, but it needs to be the ID of the _survey_ as in the list of questions, which I think would be `surveyVisit.survey_response.survey_id` here.

Reply:
I need learn to keek all this servey names straight. Ok I didn't see anything at the time that was close to matching Id and thought that what was ment. Though it was in the name 'surveyId'. I pushed up that change.